src/views/studio/studio-comments.jsx

```diff
@@ -74,54 +75,67 @@ const StudioComments = ({
                 <h2><FormattedMessage id="studio.commentsHeader" /></h2>
                 {canEditCommentsAllowed && <StudioCommentsAllowed />}
             </div>
```

Comment:
This part of the diff is much easier to read if you turn whitespace changes off.